roles/device_roles/adprosilica/templates/base.cmd.j2

```diff
@@ -0,0 +1,29 @@
+errlogInit(20000)
+
+dbLoadDatabase("{{ deploy_ioc_required_module_path }}/dbd/prosilicaApp.dbd")
+
+prosilicaApp_registerRecordDeviceDriver(pdbbase)
+
+prosilicaConfig("$(PORT)", "$(CAMERA_ID)", 50, 0)
+
+asynSetTraceIOMask("$(PORT)",0,2)
+
+dbLoadRecords("$(ADPROSILICA)/db/prosilica.template","P=$(PREFIX),R=cam1:,PORT=$(PORT),ADDR=0,TIMEOUT=1")
+
+# Create a standard arrays plugin, set it to get data from first Prosilica driver.
+NDStdArraysConfigure("Image1", 5, 0, "$(PORT)", 0, 0)
+
+# Use this line if you want to use the Prosilica in 8,12 or 16-bit modes.
+# It uses an 16-bit waveform record, so it uses twice the memory and bandwidth required for only 8-bit data.
+dbLoadRecords("$(ADCORE)/db/NDStdArrays.template", "P=$(PREFIX),R=image1:,PORT=Image1,ADDR=0,TIMEOUT=1,NDARRAY_PORT=$(PORT),TYPE=Int16,FTVL=SHORT,NELEMENTS=4177920")
+
+# Load all other plugins using commonPlugins.cmd
+< $(ADCORE)/iocBoot/commonPlugins.cmd
+set_requestfile_path("$(EPICS_BASE)/req")
+
+#asynSetTraceMask("$(PORT)",0,255)
+
+iocInit()
```

Comment:
```suggestion
```

Reply:
This is no longer needed, remove this block in future roles